# Enable Mediapipe Solutions
enable_right_hand: true
enable_left_hand: true
enable_pose: true
enable_face: false

# Training Parameters
min_epochs: 100
max_epochs: 2000
patience: 50
batch_size: 64
optimizer: Adam
learning_rate: 1e-3
loss_function: cross_entropy
train_set_size: 0.7
validation_set_size: 0.15
test_set_size: 0.15

# Torch Compilation
compilation_mode: default
torch_compilation: false
profiler: false
fast_dev_run: false

hydra:
  run:
    dir: 
      /home/davide/ROS/multimodal_fusion_ws/src/Mediapipe/mediapipe_gesture_recognition/data/outputs/${now:%Y-%m-%d}/${now:%H-%M-%S}
